apps/emqx_management/test/emqx_mgmt_api_clients_SUITE.erl

```diff
@@ -715,6 +715,215 @@ t_query_clients_with_time(_) ->
     {ok, _} = emqx_mgmt_api_test_util:request_api(delete, Client1Path),
     {ok, _} = emqx_mgmt_api_test_util:request_api(delete, Client2Path).
 
+t_query_multiple_clients(_) ->
```

Comment:
maybe add a test case to cover the kind of clientIDs which require url encoding.

Reply:
Done